wandb/integration/keras/callbacks/model_checkpoint.py

```diff
@@ -171,3 +181,10 @@ def _check_filepath(self) -> None:
                 "This ensures correct interpretation of the logged artifacts.",
                 repeat=False,
             )
+
+    def _patch_tf_keras_version(self) -> None:
+        from pkg_resources import parse_version
+
+        self.is_old_tf_keras_version: bool = parse_version(
```

Comment:
Please init this attribute in __init__ (initializing class vars outside of __init__ is an anti-pattern).